steps/generate-diff-image/0.0.1/step.yml

```diff
@@ -0,0 +1,55 @@
+title: generate-diff-image
```

Comment:
Note:
I would go with the `Generate Image Difference` for the title.
What do you think?

Reply:
Thank you for the review! I renamed step and recreated Pull Request.
https://github.com/bitrise-io/bitrise-steplib/pull/2090